result/lldp.yaml

```diff
@@ -0,0 +1,90 @@
+openapi: 3.1.0
+
+info:
+  title: LLDP results models
+  version: ^0.0.0
+
+components:
+  schemas:
+    Lldp.Metrics.Request:
+      description: >-
+        The request to retrieve LLDP per interface metrics/statistics.
+      type: object
+      properties:
+        lldp_names:
+          description: >-
+            The names of LLDP interfaces to return results for.
+            An empty list will return results for all LLDP interfaces.
+          type: array
+          items:
+            type: string
+          x-constraint:
+            - "/components/schemas/Device.Lldp/properties/name"
+        column_names:
+          description: >-
+            The requested list of column names for the result set.
+            If the list is empty then metrics for all columns will be returned.
+            The name of LLDP interface can not be excluded.
+          type: array
+          items:
+            type: string
+            enum:
+              - frames_in
+              - frames_out
+              - frames_error_in
+              - frames_discard
+              - tlvs_discard
+              - tlvs_unknown
+              - frames_error_out
+              
+    Lldp.Metric:
+      description: >-
+        LLDP per interface statistics information.
+      type: object
+      properties:
+        name:
+          description: >-
+            The name of the configured LLDP interface.
+          type: string
+        frames_in:
+          description: >-
+            Number of LLDP frames received.
+          type: integer
+          format: int64
+          minimum: 0
+        frames_out:
+          description: >-
+            Number of LLDP frames transmitted.
+          type: integer
+          format: int64
+          minimum: 0
+        frames_error_in:
+          description: >-
+            Number of LLDP frames received with packet errors.
+          type: integer
+          format: int64
+          minimum: 0
+        frames_discard:
+          description: >-
+            Number of LLDP frames received that are discarded.
+          type: integer
+          format: int64
+          minimum: 0
+        tlvs_discard:
```

Comment:
Not sure what this count would give us . Is it even needed in first revision ?  How does an user know what TLVs we are accepting and what we are discarding . As a test tool, we probably just receive TLVs, store the ones we need to show in Learned Info and drop the rest anyway.  Unless we specify exactly when this counter would increase, looks like a problematic stat to implement and interpret.

Reply:
added description